python/grass/temporal/temporal_algebra.py

```diff
@@ -795,7 +795,8 @@ def __init__(
         )
         self.process_chain_dict = (
             {}
-        )  # This dictionary stores all processes, as well as the maps to register and remove
+        )  # This dictionary stores all processes, as well as the maps to register and
+        # remove
```

Comment:
A long comment like this one should go before the variable. This will remove the need for variables around it. You can fix the following variables in the same way (lines 800 and 803).

Reply:
There were a few like this. Should I go back and change them like this?

Reply:
Yes, that would be ideal. I mean in the case that the comment line is now split over multiple lines. If you catch and fix the other case when the comment causes the parentheses to appear, that would be great, but not needed in this PR.

Reply:
I made changes to this file. I took a note to remind me to fix something like this if I see it again.  I will submit another PR with these changes later.

Reply:
Sounds good!